### PR TITLE
[serverless] Adding env variables to support .net and java instruments

### PR DESCRIPTION
**Pull request by nine5two7**
What does this PR do?
Setting env variables in the dd_wrapper file to support .net and java instruments.
This is the step  1.
Step 2 is to update the corresponding datadog-ci and serverless plugin.

Motivation:
https://datadoghq.atlassian.net/browse/SLS-2190

Describe how to test/QA your changes
Manual e2e test

### CODE REVIEW

File: scripts/datadog_wrapper.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```
Comment (tianchu) [inline scripts/datadog_wrapper.sh:1]: Can we remove the `.sh` extension from the wrapper? the examples from AWS are all without the .sh extension, i think one benefit is that we could potentially change it to a python or non-sh script down the road without asking users to update their configurations.

Comment (agocs) [reply scripts/datadog_wrapper.sh:1]: I agree it Should Be Fine to remove .sh as long as we have the shebang at the beginning of the file

File: scripts/datadog_wrapper.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  export AWS_LAMBDA_RUNTIME_API="127.0.0.1:9000"
+
+  if [ "$DD_LOG_LEVEL" == "debug" ]
+  then
+    echo "[bootstrap] rerouting AWS_LAMBDA_RUNTIME_API to $AWS_LAMBDA_RUNTIME_API"
+  fi
+fi
+
+# if it is .Net
+echo "The runtime is $AWS_EXECUTION_ENV"
```
Comment (maxday) [inline scripts/datadog_wrapper.sh:24]: I think this should be protected by a if DD_LOG_LEVEL = debug ?

File: scripts/datadog_wrapper.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+if [[ "$AWS_EXECUTION_ENV" == *"dotnet"* ]];
+then
+  echo "Configuring for the .NET runtime!"
```
Comment (maxday) [inline scripts/datadog_wrapper.sh:27]: same here

File: scripts/datadog_wrapper.sh [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  export CORECLR_ENABLE_PROFILING="1"
+  export CORECLR_PROFILER="{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
+  export CORECLR_PROFILER_PATH="/opt/datadog/Datadog.Trace.ClrProfiler.Native.so"
+  export DD_DOTNET_TRACER_HOME="/opt/datadog"
+fi
+
+# if it is java
+if [[ "$AWS_EXECUTION_ENV" == *"java"* ]];
+then
+  echo "Configuring for the Java runtime!"
```
Comment (maxday) [inline scripts/datadog_wrapper.sh:37]: same here

**Review by maxday — COMMENTED** [review]
Looks good! I left some minor comments

File: scripts/datadog_wrapper.sh (hunk 1 above)
Comment (nine5two7) [reply scripts/datadog_wrapper.sh:1]: I will create a new revision for it.

File: scripts/datadog_wrapper.sh (hunk 2 above)
Comment (nine5two7) [reply scripts/datadog_wrapper.sh:24]: Will fix it in the next revision

File: scripts/datadog_wrapper.sh (hunk 3 above)
Comment (nine5two7) [reply scripts/datadog_wrapper.sh:27]: Will fix it in the next revision

**Review by maxday — APPROVED** [review]

**Pull request merged**